src/app/pages/news/11.3/02-inscription-view.md

```diff
@@ -1,6 +1,22 @@
 ## Inscription View {#inscriptionView}
 
-Massive design update, browser, finalization ...
+The new Inscription View is finally finished! 
+We've not only successfully migrated all the old SWT-based 
+masks and browsers into the new world but have also given the entire 
+interface an amazing redesign.
+
+- **UI update**:  Prepare to be amazed by the refreshed user interface! 
+Thanks to the sleek new design, navigating through inscriptions has never been 
+smoother, making oversight effortless. Among other things, the UI update features
+ collapsible sections for every subpart, improved tab naming and overall interface streamlining.
+- **Browsers**: The Browsers, a familier tool known from the LTS, have now 
+successfully transitioned into the new Inscription View. They stand ready to assist 
+as before, helping you swiftly locate and insert functions, types, 
+attributes, CMS entries and more.
+- **Quick Completion**: A notable improvement in this version is the completion 
+list within script/code fields. Users now have an even larger and better 
```

Comment:
I wouldn't talk about a third-party "users": but directly address the user which is reading this statement e.g.

> You now get more functions, types and attributes auto inserted whenever you ask for it.